content/ru/index.md

```diff
@@ -217,7 +217,7 @@ document.body
 </code></pre>
 
 <div class="home-demo">
-<github-stars simple user="preactjs" repo="preact"></github-stars>
+<github-stars user="preactjs" repo="preact"></github-stars>
```

Comment:
Missed in #1067, however, it looks _a lot_ better IMO? I don't see a need for `simple`.

| Before / 'simple' Stars | After|
| --- | --- |
| ![Simple Stars](https://github.com/preactjs/preact-www/assets/33403762/1a81dcb1-47d6-4fa6-a7cc-12f6df50b5ce) | ![temp2](https://github.com/preactjs/preact-www/assets/33403762/ff304b6d-37be-4151-b2d7-73dece9e60d1) |

Reply:
TODO: Example code needs to be updated too